### DISCUSSION TITLE
Is it a good idea to add statistical sampling method in this crate like "pytorch"?

### DISCUSSION THREAD

**Discussion by dbsxdbsx** [neuronika/neuronika#102] [Ideas]
I am new to this crate, and I do appreciated that it so compact but useful like `pytorch`, but just one thing missing: the distribution feature, like [this](https://pytorch.org/docs/stable/distributions.html). 

By the way, I am using https://github.com/LaurentMazare/tch-rs with [another Ad-hoc distribution crate](https://github.com/spebern/tch-distr-rs), And here, Do you think it preferred to add this kind of features directly in  `neuronika` ?  Personally, I prefer add this feature as an "addiitonal" feature for this project/crate, so that user can decided whether to use it or not by easy configuration  in toml.

**Comment by frjnn** [comment]
Yes, we can add it as another member of the workspace in the new [major release](https://github.com/neuronika/neuronika/tree/variable-rework). I'm not a sampling expert and I'm currently working on GPU support so I may not focus on it for some time. Ideas & PRs welcome.
